src/components/Accordion.tsx

```diff
@@ -0,0 +1,142 @@
+import { Css, Icon, useTestIds, Palette } from "src/index";
+import { Dispatch, ReactNode, SetStateAction, useEffect, useState } from "react";
+import { useFocusRing } from "react-aria";
+import { useId } from "@react-aria/utils";
+
+export interface AccordionProps {
+  label?: string;
```

Comment:
**question:** Is `label` necessary? I see we're only using it to provide a default testId. I would prefer we just remove this and set the testids hook as `useTestIds(props, "accordion");`